### ISSUE TITLE
Android library adds required feature "android.hardware.location.gps"

### ISSUE THREAD

**Issue by yperess** [mapbox/mapbox-gl-native#10345]
**Platform: Android**
**Mapbox SDK version: 5.2.0-beta.2**

### Steps to trigger behavior
 1. Include mapbox in dependencies
 2. Build

### Expected behavior
The resulting merged AndroidManifest.xml should not require the feature android.hardware.location.gps instead it should use `required="false"`. This took a while to find and teams wanting to port their apps to Chromebooks will all have to override this value.

### Actual behavior
The entry below is added to the AndroidManifest.xml:

    <uses-feature name="android.hardware.location.gps" required="true"/>

**Comment by tobrun** [comment]
Thank you for the report @yperess, we don't add this value ourselves but this value will probably come from another dependency. We will work on overriding this value in our AndroidManifest.xml file in our SDK.

**Comment by yperess** [comment]
Awesome! Thanks for the quick response!

**Comment by zugaldia** [comment]
While this lands, a workaround is to manually add the following line to your app's Manifest to remove this requirement:

```
<uses-feature android:name="android.hardware.location.gps" android:required="false" tools:replace="required" />
```

**Comment by yperess** [comment]
Yup, that's correct. I just figured adding this fix will make it easier for others and save them the time of hunting down the source of the requirement.

**Comment by tobrun** [comment]
Landed with #10347
